linkedList/doublyLL.js

```diff
@@ -0,0 +1,94 @@
+class DoublyLinkedList {
+    constructor(value) {
+        this.head = {
```

Comment:
there could be empty linked list as well, this is not possible with your implementation